specification/examples/model-walkthrough_smdl.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Name": "Simon IvC",
+  "BIDSModelVersion": "1.0.0",
+  "Input": {"subject": ["01", "02", "03"], "task": "simon"},
+  "Nodes": [
+    {
+      "Level": "Run",
+      "Name": "run_level",
+      "GroupBy": ["run", "subject"],
+      "Model": {"X": [1, "incongruent", "congruent"], "Type": "glm"},
+      "Contrasts": [
+        {
+          "Name": "IvC",
+          "ConditionList": ["incongruent", "congruent"],
+          "Weights": [1, -1],
+          "Test": "t"
+        }
+      ]
+    },
+    {
+      "Level": "Subject",
+      "Name": "subject_level",
+      "GroupBy": ["subject", "contrast"],
+      "Model": {"X": [1], "Type": "meta"},
+      "DummyContrasts": {"Test": "t"}
+    },
+    {
+      "Level": "Dataset",
+      "Name": "one-sample_dataset",
```

Comment:
In case it wasn't clear from the other [comment](https://github.com/bids-standard/stats-models/pull/37/files#r879962947), I would rename this to 'dataset_level'